dotcom-rendering/src/web/components/DynamicPackage.tsx

```diff
@@ -71,39 +46,12 @@ export const DynamicPackage = ({
 								}
 								padBottomOnMobile={false}
 							>
-								<Card
+								<FrontCard
+									trail={card}
 									containerPalette={containerPalette}
-									containerType="dynamic/package"
 									showAge={showAge}
-									linkTo={card.url}
-									format={card.format}
-									headlineText={card.headline}
 									headlineSize="medium"
```

Comment:
Medium is the default size I think, shouldn't need to specify it.

Reply:
Ah, cool. Thanks, I'll update